e2e/pages/ShowPage.js

```diff
@@ -76,18 +76,17 @@ export default (url, initialField = 'title') => driver => ({
         return driver.sleep(200);
     },
 
-    delete() {
-        return driver
-            .findElement(this.elements.deleteButton)
-            .then(button => button.click())
-            .then(() =>
-                driver.findElement(this.elements.deleteConfirmButton).click()
-            )
-            .then(() => driver.sleep(200))
-            .then(() =>
```

Comment:
I prefer that you don't remove this part, as it's used to avoid that the snackbar is still over the create button in further tests

Reply:
The notification for a successful delete has disappeared. WIll be addressed in another PR.